datafusion/src/optimizer/utils.rs

```diff
@@ -228,6 +228,13 @@ pub fn from_plan(
                 name: name.clone(),
             })
         }
+        LogicalPlan::DropMemoryTable { name, if_exist, .. } => {
+            Ok(LogicalPlan::DropMemoryTable {
```

Comment:
I think this should handled the same as cases without input, like `LogicalPlan::EmptyRelation`